docs/mkdocs.yml

```diff
@@ -22,69 +22,79 @@ plugins:
-    - partitioning.md
-    - performance.md
-    - reliability.md
-    - schemas.md
-  - Views:
-    - view-configuration.md
-  - Spark:
-    - spark-getting-started.md
-    - spark-configuration.md
-    - spark-ddl.md
-    - spark-procedures.md
-    - spark-queries.md
-    - spark-structured-streaming.md
-    - spark-writes.md
-  - Flink:
-    - flink.md
-    - flink-connector.md
-    - flink-ddl.md
-    - flink-queries.md
-    - flink-writes.md
-    - flink-actions.md
-    - flink-configuration.md
-  - hive.md
-  - Trino: https://trino.io/docs/current/connector/iceberg.html
-  - Daft: daft.md
-  - Firebolt: https://docs.firebolt.io/reference-sql/functions-reference/table-valued/read_iceberg
-  - Estuary: https://docs.estuary.dev/reference/Connectors/materialization-connectors/apache-iceberg/
-  - Tinybird: https://www.tinybird.co/docs/forward/get-data-in/table-functions/iceberg
-  - Redpanda: https://docs.redpanda.com/current/manage/iceberg/about-iceberg-topics
-  - RisingWave: risingwave.md
-  - ClickHouse: https://clickhouse.com/docs/en/engines/table-engines/integrations/iceberg
-  - Presto: https://prestodb.io/docs/current/connector/iceberg.html
-  - Dremio: https://docs.dremio.com/data-formats/apache-iceberg/
-  - Starrocks: https://docs.starrocks.io/en-us/latest/data_source/catalog/iceberg_catalog
-  - Amoro: amoro.md
-  - Amazon Athena: https://docs.aws.amazon.com/athena/latest/ug/querying-iceberg.html
-  - Amazon EMR: https://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-iceberg-use-cluster.html
-  - Amazon Data Firehose: https://docs.aws.amazon.com/firehose/latest/dev/apache-iceberg-destination.html
-  - Amazon Redshift: https://docs.aws.amazon.com/redshift/latest/dg/querying-iceberg.html
-  - Google BigQuery: https://cloud.google.com/bigquery/docs/iceberg-tables
-  - Snowflake: https://docs.snowflake.com/en/user-guide/tables-iceberg
-  - Impala: https://impala.apache.org/docs/build/html/topics/impala_iceberg.html
-  - Databend: https://docs.databend.com/guides/access-data-lake/iceberg
-  - Doris: https://doris.apache.org/docs/dev/lakehouse/catalogs/iceberg-catalog
-  - Druid: https://druid.apache.org/docs/latest/development/extensions-contrib/iceberg/
-  - DuckDB: https://duckdb.org/docs/preview/core_extensions/iceberg/overview
-  - Kafka Connect: kafka-connect.md
-  - Integrations:
-    - aws.md
-    - dell.md
+  - Concepts:
+    - Tables:
+      - branching.md
+      - configuration.md
+      - evolution.md
+      - maintenance.md
+      - metrics-reporting.md
+      - partitioning.md
+      - performance.md
+      - reliability.md
+      - schemas.md
+    - Views:
+      - view-configuration.md
+  - Libraries:
+    - Java:
+      - Quickstart: java-api-quickstart.md
+      - API: api.md
+      - Javadoc: ../../javadoc/latest/
+    - PyIceberg: https://py.iceberg.apache.org/
+    - IcebergRust: https://rust.iceberg.apache.org/
+    - IcebergGo: https://pkg.go.dev/github.com/apache/iceberg-go/
+  - Clients:
```

Comment:
should this be called `Integrations` or `Engines`? 

-  I know `Integrations` was previously used for aws, azure storage etc. I like the new name of `Storage` for those. 
- `Integrations` here can mean integrations with engines

I am a little reserved about `Clients`, because people probably can refer PyIceberg as Python client.

interested to hear peoples' thoughts.

Reply:
I agree with `Integrations`. 
ff8644f8c6472bb4f015bb20b31fb8024715bf56